utils/ExampleRunner/example-runner-cli.js

```diff
@@ -20,6 +20,7 @@ const rootPath = path.resolve(path.join(__dirname, '../..'));
 program
   .option('-c, --config [file.js]', 'Configuration file')
   .option('--no-browser', 'Do not open the browser')
+  .option('--https', 'Enable https')
```

Comment:
Adding a --https option to allow running examples over https with a self signed cert.